frontend/public/components/container-selector.tsx

```diff
@@ -8,7 +8,7 @@ export const ContainerSelector: React.FC<ContainerSelectorProps> = ({
   onChange,
   selected,
 }) => (
-  <div>
+  <div className="pf-c-form__checkbox--row">
```

Comment:
Nit:  use `pf-c-form__checkbox-row` since you're not modifying an existing `pf-c-form__checkbox` class.

Reply:
updated